src/dfcx_scrapi/tools/nlu_utils.py

```diff
@@ -0,0 +1,329 @@
+"""Utility file for NLU Utterance Similarity to work with CX resources."""
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pandas as pd
+import scann
```

Comment:
Calling out a dependency risk with `scann` due to the lack of distribution outside of Linux distros.  
I know this was originally designed to use on Google Colab or Jupyter Notebook instances, however it will limit users that are trying to run on Mac, Windows, etc.

No action at this time, but we should aim to replace this and it's subsequent methods with something more universally distributed.